arangod/VocBase/vocbase.h

```diff
@@ -397,4 +409,11 @@ void TRI_SanitizeObject(arangodb::velocypack::Slice const slice,
 void TRI_SanitizeObjectWithEdges(arangodb::velocypack::Slice const slice,
                                  arangodb::velocypack::Builder& builder);
 
+namespace arangodb {
+using VocbaseOptionsTuple = std::tuple<std::string /*sharding*/, std::uint32_t /*replication*/, std::uint32_t /*minReplication*/>;
```

Comment:
Can we use a struct here? It will give more semantic access to the members (e.g. `options.sharding`) and be more easily amended if we need additional options in the future.

Reply:
Definitely +1 for a struct!
As there is a struct defined below, do we still need this extra definition for the tuple?